easystacks/habrok-2023.01-cvmfs.yml

```diff
@@ -409,3 +409,5 @@ easyconfigs:
   - QuantumESPRESSO-7.3.1-foss-2023a.eb
   - MUSCLE-3.8.31-GCCcore-11.2.0.eb
   - SAMtools-0.1.19-GCC-10.3.0.eb
+  - COMSOL-6.0.eb
```

Comment:
COMSOL 6.0 is listed already in the `generic` easystack, shouldn't this new one also go there instead?

Reply:
A my mistake. I've moved COMSOL 6.2 to the generic stack and removed COMSOL from the standard stack file.